doc/build_apps/api.rst

```diff
@@ -65,6 +65,12 @@ Supporting Types
 .. doxygenenum:: ccf::TxStatus
```

Comment:
TxID should be there too

Reply:
Too many branches, I thought it was - have added it now.